tests/test_physics.py

```diff
@@ -980,3 +990,232 @@ def test_articulated_object_fixed_base_proxy():
             robot.joint_positions = [0.0]
             assert robot.contact_test()
             cube_obj.motion_type = habitat_sim.physics.MotionType.DYNAMIC
+
+
+@pytest.mark.skipif(
+    not habitat_sim.built_with_bullet,
+    reason="ArticulatedObject API requires Bullet physics.",
+)
+def test_articulated_object_damping_joint_motors():
+    # test automated creation of joint motors from URDF configured joint damping values
+    robot_file = "data/test_assets/urdf/kuka_iiwa/model_free_base.urdf"
+    cfg_settings = examples.settings.default_sim_settings.copy()
+    cfg_settings["scene"] = "NONE"
+    cfg_settings["enable_physics"] = True
+    hab_cfg = examples.settings.make_cfg(cfg_settings)
+    with habitat_sim.Simulator(hab_cfg) as sim:
+        art_obj_mgr = sim.get_articulated_object_manager()
+        # parse URDF and add an ArticulatedObject to the world
+        robot = art_obj_mgr.add_articulated_object_from_urdf(filepath=robot_file)
+        assert robot.is_alive
+        # When URDF joint damping is defined, we generate a set of motors automatically
+        # get a map of joint motor ids to starting DoF indices
+        existing_joint_motors = robot.get_existing_joint_motor_ids()
```

Comment:
`robot.get_existing_joint_motor_ids()`
Could/Should this be a read-only property?